target/linux/generic/files/drivers/platform/mikrotik/rb_hardconfig.c

```diff
@@ -546,6 +355,61 @@ static int hc_wlan_data_unpack_lzor(const u16 tag_id, const u8 *inbuf, size_t in
 	return ret;
 }
 
+static int hc_wlan_data_unpack_lz77(const u16 tag_id, const u8 *inbuf, size_t inlen,
```

Comment:
Yes, absolutely. Just wanted to make sure I got LZ77 working without breaking LZOR first. Happy with `hc_wlan_data_unpack_lzor_77zl`?

---

While I'm there, as a separate commit; I also see flash.ko always skips to offset 0x67c (0x680-4) after the LZOR (with prefix) decompression, so that could save some walking? https://github.com/openwrt/openwrt/blob/646ebbd32ca4d776c64c31e85c08dc72fec25d7d/target/linux/generic/files/drivers/platform/mikrotik/rb_hardconfig.c#L517

Reply:
Until Mikrotik silently changes things and breaks everything for us. I think it's safer to keep this programmatic and the performance impact is not huge IMHO :-)